activesupport/lib/active_support/hash_with_indifferent_access.rb

```diff
@@ -316,4 +316,5 @@ def set_defaults(target) # :doc:
   end
 end
 
+# :stopdoc:
 HashWithIndifferentAccess = ActiveSupport::HashWithIndifferentAccess
```

Comment:
👍 on removing the top level `HashWithIndifferentAccess` constant.

@robin850 If there's mentions of `ActiveSupport::HWIA` in guides or elsewhere we should expand them to the correct name.